.travis.yml

```diff
@@ -18,17 +18,16 @@ matrix:
     env: TEST_TARGET=coding_standards
 
 before_install:
-    - URL=http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh
-    - wget $URL -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - conda update --yes --all
-    - conda config --add channels conda-forge --force
-    # hopefully fix issues with missing libraries and LXML
-    - conda install -y 'icu=58.*'
-    - travis_retry conda create --yes -n TEST --file requirements.txt $CONDA
-    - source activate TEST
-    - travis_retry conda install --yes pytest flake8
+  - wget http://bit.ly/miniconda -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+  - conda update conda
+  - conda config --add channels conda-forge --force
+  # hopefully fix issues with missing libraries and LXML
+  - conda install -y 'icu=58.*'
```

Comment:
Was necessary to add this to keep LXML from breaking due to a missing library.  Might be worth looking at the latest LXML package in conda to see if it's an issue.

Reply:
This is certainly a bad dependency package on the conda side and we should not do that here.
I guess that is happening only with Python 3.4, which we dropped in #41, but if you do observe this in other Pythons then we should report it back to conda-forge.